apps/glusterfs/db_operations.go

```diff
@@ -452,3 +452,292 @@ func DeleteBricksWithEmptyPath(db *bolt.DB, all bool, clusterIDs []string, nodeI
 	}
 	return nil
 }
+
+func deleteChangeOwnerEntry(tx *bolt.Tx, action PendingOperationAction, dryRun bool) error {
+	switch {
```

Comment:
Since every case in this switch statement compares to action.Change you could have just writen `switch action.Change {` and `case OpAddBrick` etc. But this works, this is mostly just me nitpicking more.

Reply:
Thanks! I remember looking at it and feeling something odd. That is surely more idiomatic.